packages/ibm-products/src/components/UserAvatar/UserAvatar.js

```diff
@@ -0,0 +1,166 @@
+const componentName = 'UserAvatar';
+
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+// Default values can be included here and then assigned to the prop params,
+// e.g. prop = defaults.prop,
+// This gathers default values together neatly and ensures non-primitive
+// values are initialized early to avoid react making unnecessary re-renders.
+// Note that default values are not required for props that are 'required',
+// nor for props where the component can apply undefined values reasonably.
+// Default values should be provided when the component needs to make a choice
+// or assumption when a prop is not supplied.
+
+// Default values for props
+// const defaults = {
+//   /* TODO: add defaults for relevant props if needed */
+// };
+
+/**
+ * TODO: A description of the component.
+ */
+
+const defaults = {
+  renderIcon: () => <User size={32} />,
+  tooltipAlignment: 'bottom',
+  tooltipText: 'Iam tooltip',
+};
+
+export let UserAvatar = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+      backgroundColor,
+      className,
+      /* TODO: add other props for UserAvatar, with default values if needed */
+      renderIcon = defaults.renderIcon,
+      tooltipText = defaults.tooltipText,
+      tooltipAlignment = defaults.tooltipAlignment,
+      // Collect any other property values passed in.
+      ...rest
+    },
+    ref
+  ) => {
+    const carbonPrefix = usePrefix();
+    const icons = {
+      user: {
+        md: <User size={32} />,
+      },
+      group: {
+        md: <Group size={32} />,
+      },
+    };
+    const getItem = (renderIcon) => {
+      if (renderIcon === User) {
+        return icons.user['md'];
+      } else if (renderIcon === Group) {
+        return icons.group['md'];
+      } else {
+        return renderIcon;
+      }
+    };
+
+    const SetItem = getItem(renderIcon);
+
+    const renderUserAvatar = () => (
+      <div
+        {
+          // Pass through any other property values as HTML attributes.
+          ...rest
+        }
+        tabIndex={0}
+        className={cx(
+          blockClass, // Apply the block class to the main HTML element
+          className, // Apply any supplied class names to the main HTML element.
+          `${blockClass}--${backgroundColor}`,
+          // example: `${blockClass}__template-string-class-${kind}-n-${size}`,
+          {
+            // switched classes dependant on props or state
+            // example: [`${blockClass}__here-if-small`]: size === 'sm',
+          }
+        )}
+        ref={ref}
+        role="button"
+        {...getDevtoolsProps(componentName)}
+      >
+        <SetItem />
+      </div>
+    );
+
+    return (
+      SetItem &&
+      (tooltipText ? (
+        <Tooltip
+          align={tooltipAlignment}
+          label={tooltipText}
+          className={`${blockClass}__tooltip ${carbonPrefix}--icon-tooltip`}
```

Comment:
I wonder if we need to continue overriding the default tooltip behavior in this case. The default tooltip did update in Carbon 11. Maybe it’s worth starting with the new size before overriding it? Can ask @marion-bruells and @cate-wilcox for feedback on this.


In Carbon 11, the default tooltip size is a bit wider. We can override this to follow the original size 

| No overrides | Original user profile image |
---------------|----------------------------
<img width="210" alt="image" src="https://github.com/carbon-design-system/ibm-products/assets/1717398/a1ca8660-6b8a-4d37-91be-63fcae1c1e47"> | <img width="203" alt="image" src="https://github.com/carbon-design-system/ibm-products/assets/1717398/0e5f10d0-665d-4d62-919d-e1926aa3863b">

Reply:
I pinged them in slack. Waiting for the response from @marion-bruells and @cate-wilcox.